content/applications/finance/documents/actions.rst

```diff
@@ -0,0 +1,117 @@
+   :align: center
+   :alt: Workflow actions smart button in Odoo Documents
+
+.. note::
+   An action applies to all *Child Workspaces* under the *Parent Workspace* you selected (i.e. to
+   all sub-folders under the selected folder).
+
+.. tip::
+   If you use the :doc:`developer mode <../../general/developer_mode/activate>`, you can access
+   directly all your actions by going to :menuselection:`Configuration --> Actions`.
+
+Set the conditions
+------------------
+
+After having named your workflow action, you can set the conditions that trigger the appearance
+of the action button on the right-side panel when selecting a file.
+
+There are three basic types of conditions you can set:
+
+#. **Tags**: you can both use the *Contains* and *Does not contain* conditions, meaning the files
+   *must have* or *mustn't have* the tags you set here.
+
+#. **Contact**: the files must be associated with the contact you set here.
+
+#. **Owner**: the files must be associated with the owner you set here.
+
+.. image:: actions/workflow-action-basic-conditions.png
+   :align: center
+   :alt: Example of a workflow action's basic condition in Odoo Documents
+
+.. tip::
+   If you don't set any condition, the action button appears for all files located inside the
+   workspace you selected.
+
+Advanced condition type: domain
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. important:: It is recommended to have some knowledge of Odoo development to configure properly
+   *Domain* filters.
+
+To access the *Domain* condition, the :doc:`developer mode <../../general/developer_mode/activate>`
+needs to be activated. Once that's done, select the *Domain* condition type, and click on *Add
+Filter*.
+
+.. image:: actions/activate-domain-condition-type.png
+   :align: center
+   :alt: Activating the domain condition type in Odoo Documents
+
+To create a rule, you typically select a field, an operator, and a value. For example, if you want
+to add a workflow action to all the PDF files inside a workspace, set the field to: *Mime Type*, the
+operator to *contains*, and the value to *pdf*.
+
+.. image:: actions/domain-filter-example.png
+   :align: center
+   :alt: Example of a workflow action's domain condition in Odoo Documents
+
+Click on *Add node* (plus circle icon) and *Add branch* (ellipsis icon) to add conditions and
+sub-conditions. You can then specify if your rule should match *ALL* or *ANY* conditions. You can
+also edit the rule directly using the *Code editor*.
+
+.. image:: actions/add-branch-node-domain-condition.png
+   :align: center
+   :alt: Add a node or a branch to a workflow action's condition in Odoo Documents
+
+Configure the actions
+---------------------
+
+Select the *Actions* tab to set up your action. You can simultaneously:
+
+#. **Set Contact**: add a contact to the file, or replace an already existing contact with a new
+   one.
+
+#. **Set Owner**: add an owner to the file, or replace an already existing owner with a new one.
```

Comment:
"Already existing [insert option]" is kind of redundant. You could just say "replace an existing contact..."